parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -294,15 +283,19 @@ ensureDocker() {
             sleep 1
         fi
     done
+    systemctl is-active --quiet containerd && (systemctl_disable 20 30 120 containerd || exit $ERR_SYSTEMD_CONTAINERD_STOP_FAIL)
```

Comment:
where do we try to stop the containerd?

Reply:
just to be safe since the docker systemd unit will own its own containerd process that's not directly managed by systemd. in theory this line isn't needed if RP/agentbaker correctly uses the non-containerd VHDs.

Reply:
my overlook. we do the stopping here too.